about.html

```diff
@@ -0,0 +1,46 @@
+        <div class = "midbox">
+            <img width = "900px" height = "650px"src="_assets/IMG_2144.JPG">
+            <div style="padding-top: 60px">
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: center; padding-left: 20px; padding-right: 20px; padding-bottom: 60px;">Autoslug is a Baskin-affiliated organization dedicated to developing practical applications of artificial intelligence (AI), machine learning (ML), and computer vision (CV). We believe that these technologies have the potential to revolutionize many aspects of our lives, and we are committed to providing students with the opportunity to learn how to use them in actual settings. We believe that students should have the opportunity to learn how to use these tools in an applicable real-world environment; to ultimately expand beyond the theoretical side of these advanced concepts and explore their intersection with humans, data, and other forms of hardware.</div>
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: left; padding-left: 20px; padding-right: 20px; padding-bottom: 0px;">We offer a variety of projects and resources to help students achieve their goals and develop skills in AI, ML, and CV, including:</div>
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: left; padding-left: 20px; padding-right: 20px; padding-bottom: 0px;">Projects: We encourage students to participate in our hands-on projects, each applying several core software & control concepts to solve real-world problems.</div>
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: left; padding-left: 20px; padding-right: 20px; padding-bottom: 0px;">Research opportunities: We offer students the opportunity to work on cutting-edge research projects with on-campus labs.</div>
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: left; padding-left: 20px; padding-right: 20px; padding-bottom: 0px;">Workshops: We offer a variety of workshops, tutorials, and other resources to gain experience in related topics.</div>
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: left; padding-left: 20px; padding-right: 20px; padding-bottom: 60px;">We believe that Autoslug is an essential resource for students who are interested in learning about and using AI, ML, and CV. We are committed to providing students with the opportunity to learn and grow in this exciting field.</div>
+            </div>
+    </div>
```

Comment:
the team and projects page both have a darker purple bar at the bottom, can you do that for this page too?

Reply:
So I'm not sure how exactly to explain this, but when you try to scroll down further on the team and projects page, below the bottomTile is a darker purple background (it's not exactly the same color as the bottomTile, but close). However, when you try to scroll past the bottom of the about page, the background is the same brighter purple as the rest of the page. I think this might just be a case of needing to make the css/style settings for the bar exactly the same, but see if you can reproduce this?